test/swarming/bot-harness.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
```

Comment:
Just checking that it's ok to have different Python version requirements in a single codebase

Reply:
Good point, but I am not aware of any other Python file in the codebase, except the config file for MacOS build (`kokoro/macos/dmg-settings.py`) ?

Reply:
Bazel depends on Python 2.7 IIRC